pkg/api/config.go

```diff
@@ -71,7 +71,6 @@ type Config struct {
 	LoggingWorkspace           string // workspace for Azure Log Analytics resource
 
 	// DNS configurables
-	RouterLBCNamePrefix string
```

Comment:
It is being derived now. Any data which is the direct derivative result of other configuration items in the config is overpopulating our config file. This was one of those cases (same as MasterLBPrefix but we can remove it now). 

It just cleans our internal config little bit. Agree, this could be follow-up PR, but as we discussed with @jim-minter that this could be next candidate to move to `derivative data` m,odel I just threw it here.